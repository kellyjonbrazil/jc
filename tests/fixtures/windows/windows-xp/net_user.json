{
  "domain": "",
  "user_accounts": [
    {
      "user_name": "Administrator"
    },
    {
      "user_name": "Guest"
    },
    {
      "user_name": "SUPPORT_3556336b0"
    }
  ],
  "account_origin": "\\\\DESKTOP-WINXP"
}
