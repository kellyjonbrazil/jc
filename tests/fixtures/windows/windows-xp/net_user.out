
User accounts for \\DESKTOP-WINXP

-------------------------------------------------------------------------------
Administrator            Guest                    SUPPORT_3556336b0
The command completed successfully.